### DISCUSSION TITLE
How would I use the cached version of the Az cmdlet versus the 2.1.0 that is already being used.

### DISCUSSION THREAD

**Discussion by devopsaz-owner** [actions/virtual-environments#9207] [General questions]
I tried researching but I found the following:

```
    Save-Module -Path "$psModulePath\az_$latestModVersion" -RequiredVersion $latestModVersion -Name Az -Force -ErrorAction SilentlyContinue -Verbose
    $env:PSModulePath = "$psModulePath\az_$latestModVersion`;$($env:PSModulePath)"
    Import-Module Az -ErrorAction SilentlyContinue -Verbose
```

Which seemed to work on one of my build servers,  but failed on another. Is there a way that I am supposed to be doing that I am just not?

**Comment by shamil-mubarakshin** [comment]
Hey @DevOpsAzurance, not sure if I understood your question correctly. 
Is this something `runner-images` related? In case it is, could you please provide more information and some context.

**Reply by devopsaz-owner** [reply]
Yes, this is for the runner-images.  I am trying to find out the right way to use a cached version of the tools, eg another version of Az.  I have tried the above, but I am finding different behavior depending on the runner agent and thus causing some issues.
